features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Allow checking out previous git branch to work correctly after running a Git Town commmand that deletes the current branch
```

Comment:
Feature: Allow checking out **the correct** previous **G**it branch ~~to work correctly~~ after running a Git Town commmand that deletes the current branch